packages/pds/src/services/moderation/index.ts

```diff
@@ -170,7 +170,8 @@ export class ModerationService {
     }
 
     return await builder
-      .selectAll()
+      .innerJoin('did_handle', 'did_handle.did', 'moderation_report.subjectDid')
```

Comment:
I believe we may need to make this a left join so that we can serve reports for contents that have been deleted—the inner join will omit results where there's not a matching row in `did_handle`.  In those cases we might not have the handle of the account on hand anymore, and I think we'll just have to omit the optional `subjectRepoHandle` field.